build.gradle

```diff
@@ -7,9 +7,11 @@ apply plugin: 'signing'
 
 repositories {
     jcenter()
-    jcenter{
-    	url 'http://oss.jfrog.org/artifactory/oss-snapshot-local'
-    }
+    //Commented this out, neither the SDK nor the Core libraries specify this
```

Comment:
Let's leave this in. Someone added this intentionally. It is worth a quick investigation to understand what this means.